README.md

```diff
@@ -54,50 +47,72 @@ The 2010 census tracts are used (redrawn every 10 years)
-	- Rename the relevant CSV as "[state]\_[yr]\_[income|race].csv" and save it in the /data/[state]/ directory
-		- e.g. 2011 income data for MA from ACS 5-year estimates is saved to "/data/ma/ma_2011_income.csv"
 
-- Process all of the csv data files for the state into one file by running /scripts/census_data_processing (with jupyter)
+- Download the data by going to https://factfinder.census.gov/faces/nav/jsf/pages/download_center.xhtml (OMG this site has some _wild_ 90's era graphics)
+  - Choose "I know the dataset or table(s) that I want to download."
+  - Select "American Community Survey"
+  - And then for **each year** of interest download data for 5-year estimates (e.g. "2014 ACS 5-year estimates")
+    - years of interest: beginning of bike sharing program to most recent
+  - Select a geographic type: "Census Tract - 140"
+    - be careful not to select "Census Tract (or part) - 080"
+  - Select state --> Select all Census Tract within [state] --> Add to selection --> Next
+  - Select tables (can use search) and download CSVs for tables 1 by 1 (tedious)
+    - race as "RACE" (ID:B02001)
+    - median income as "MEDIAN INCOME IN THE PAST 12 MONTHS (IN <year> INFLATION-ADJUSTED DOLLARS)" (ID:S1903)
+    - <img src="./docs/using-factfinder-ma-income.png">
+  - Download each table and open zipfile. The name of the relevant csv within looks something like "ACS*[yr]\_5YR_B02001_with_ann.csv"(race) or "ACS*[yr]\_5YR_S1903_with_ann.csv"(income) but double check the file
+  - Rename the relevant CSV as "[state]\_[yr]\_[income|race].csv" and save it in the /data/[state]/ directory
+    - e.g. 2011 income data for MA from ACS 5-year estimates is saved to "/data/ma/ma_2011_income.csv"
+
+##### New Census
 
+For state census data:
+-Download the data by going to https://data.census.gov/cedsci/table?q=american%20community%20survey
 
+- Select "Geography"
+- Then "Tract"
+- Select state --> Select all Census Tracts within [state]
+- Get to race table:
+  - Select "Topics" under Geography
+  - Go to the "Race and Ethnicity" FOlder then click the box for "Race and Ethnicity"
+  - Scroll down available datasets until B02001: RACE
+- Get to income table:
 
-### Bike Station Data
+  - Go to topics again and select Income and Poverty
+  - Select the table "S1903 | MEDIAN INCOME IN THE PAST 12 MONTHS (IN [year] INFLATION-ADJUSTED DOLLARS)"
+
+- Download the files:
+
+  - Click "DOWNLOAD TABLE", select the years you want to download, click "DOWNLOAD.CSV", and wait for prompt to say download is ready at the bottom of the screen
+  - If the downnload doesn't start up, click the download button again then a .zip file should be downloaded.
+  - The relevant file should be called "ACSDT5Y[year].[code]\_data_with_overlays_2022-04-14T225511.csv", [code] being B02001 for race and S1903 for income.
```

Comment:
file name content following `_data_with_overlays_` dependent on time of download